runtime/src/accounts.rs

```diff
@@ -815,48 +793,51 @@ impl Accounts {
         self.accounts_db.add_root(slot)
     }
 
-    fn is_non_loader_key(message: &Message, key: &Pubkey, key_index: usize) -> bool {
+    pub fn is_non_loader_key(message: &Message, key: &Pubkey, key_index: usize) -> bool {
```

Comment:
nits: Maybe marking `pub` to be used from `NonceRollbackFull::from_partial()` signifies this helper's proper place is different. Maybe time to move to `Message` (`message.rs`)?

So that, readers (including me) might help to find these related logic easily in the future: https://github.com/solana-labs/solana/pull/13868/files#r533542063

Reply:
Good point!  Yeah I'll look into relocating that one